### DISCUSSION TITLE
Disabling terminal by default when starting debug session

### DISCUSSION THREAD

**Discussion by cheeze2000** [mfussenegger/nvim-dap#754] [Q&A] [answered]
I found the `terminal_win_cmd` option that you can set up, which is by default `"belowright new"`. What command do I use if I want to permanently disable it?

**Comment by mfussenegger** [comment, marked as answer]
Some debug adapters have a `console` option with options like `integratedConsole` or `integratedTerminal`. If that's the case you switch from `integratedTerminal` to `integratedConsole`.

If the debug adapter doesn't let you control if it uses a integrated terminal you can either:

- Set `terminal_win_cmd` to something like `tabnew`, to have it open the terminal in a new tab in the background
- Set `terminal_win_cmd` to a lua function which creates a window/buffer and then hides it. Afaik nvim-dap-ui does that.
